src/MyPlot/provider/MySQLProvider.php

```diff
@@ -37,28 +38,28 @@ public function __construct(MyPlot $plugin, $cacheSize = 0, $settings) {
 		$this->settings = $settings;
 		$this->db = new \mysqli($settings['Host'], $settings['Username'], $settings['Password'], $settings['DatabaseName'], $settings['Port']);
 		$this->db->query(
-			"CREATE TABLE IF NOT EXISTS plots (id INT PRIMARY KEY AUTO_INCREMENT, level TEXT, X INT, Z INT, name TEXT, owner TEXT, helpers TEXT, denied TEXT, biome TEXT);"
+			"CREATE TABLE IF NOT EXISTS plots (id INT PRIMARY KEY AUTO_INCREMENT, level TEXT, X INT, Z INT, name TEXT, owner TEXT, helpers TEXT, denied TEXT, biome TEXT, done BOOL);"
```

Comment:
Have fun, this will not just work with old table formats. You would have to update existing tables to contain the new DONE column